packages/core/src/utils.js

```diff
@@ -7,6 +7,11 @@ import jsonpointer from "jsonpointer";
 import fields from "./components/fields";
 import widgets from "./components/widgets";
 import validateFormData, { isValid } from "./validate";
+import useEmptyValueOnChange from './hooks/useEmptyValueOnChange';
+
+export const hooks = {
```

Comment:
I just tacked the hooks onto the utils. If there is a better place, let me know.